### DISCUSSION TITLE
Access to Tapo Camera with HomeAssistant-Tapo-Control and Scrypted

### DISCUSSION THREAD

**Discussion by adriendpanopli** [JurajNyiri/HomeAssistant-Tapo-Control#791] [Q&A]
Hello there :) 

I've been trying to install the HomeAssistant-Tapo-Control plugin in HA (my tapo camera is currently running on scrypted) and I was not able to make it work ..

Failling when I have to set up the login informations (account and password)

Are the tapo camera able to run with scrypted and with the plugin ?

Thank you for your help :)

**Comment by JurajNyiri** [comment]
Hi, please post more information about what you see and what you did.
